index.html

```diff
@@ -6,17 +6,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Portfolio</title>
     <link
-      href="https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@400;500;700&family=Inter:wght@400;700&family=Poppins:wght@400;700&family=Roboto:ital,wght@0,400;0,700;0,900;1,400&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Be+Vietnam+Pro:wght@400;500;700&family=Inter:wght@400;500;700&family=Poppins:wght@400;700&family=Roboto:ital,wght@0,400;0,700;0,900;1,400&display=swap"
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <header class="header-container">
-      <nav class="logo">My Logo</nav>
-      <nav class="nav-bar">
-        <img src="images/Union.png" alt="" >
-      </nav>
-    </header>
+    <div class="container">
+      <header class="header-container">
+        <nav class="logo">My Logo</nav>
+        <nav class="nav-bar">
+          <img src="images/Union.png" alt="" />
+        </nav>
+      </header>
+      <section class="hero">
+        <h1>I’m Malik Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <article>
+            <h3>LET’S CONNECT</h3>
+            <img src="images/social media icons.png" alt="">
```

Comment:
- [ ] please this should not be an image but different social icons. they will help a user connect with you.
- [ ] kindly ensure The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)